index.js

```diff
@@ -7,8 +7,16 @@ const util = require('util');
 const Client_MySQL = require('knex/lib/dialects/mysql');
 
 class Client_MySQL_deadlock extends Client_MySQL {
+  constructor(config) {
+    super(config);
+
+    const { deadlockRetries } = config.options;
+
+    this.deadlockRetries = deadlockRetries || 5;
+  }
+
   _query(connection, obj) {
```

Comment:
not utilizing `connection` either, both are passed through to the parent class via `arguments`, but I'd keep them to have some clarity on the method signature.